python/rikai/spark/functions/vision.py

```diff
@@ -14,8 +14,10 @@
 
 """Vision related Spark UDFs.
 """
+from typing import Union
 
 # Third Party
+import os
```

Comment:
This import should be in the standard library section?

Reply:
done